zcash_client_backend/src/data_api.rs

```diff
@@ -1317,10 +1317,10 @@ pub struct SentTransaction<'a, AccountId> {
     tx: &'a Transaction,
     created: time::OffsetDateTime,
     account: AccountId,
-    outputs: Vec<SentTransactionOutput<AccountId>>,
+    outputs: &'a Vec<SentTransactionOutput<AccountId>>,
     fee_amount: NonNegativeAmount,
     #[cfg(feature = "transparent-inputs")]
-    utxos_spent: Vec<OutPoint>,
+    utxos_spent: &'a Vec<OutPoint>,
```

Comment:
TODO: check whether the `outputs()` and `utxos_spent` methods still need `as_ref()`.